helper/linux/writejob.cpp

```diff
@@ -135,17 +138,17 @@ bool WriteJob::writeCompressed(int fd) {
         return false;
     }
 
-    strm.next_in = inBuffer;
+    strm.next_in = reinterpret_cast<uint8_t*>(inBuffer);
```

Comment:
Moved the buffer to char even though uint8_t could be used too since it's implemented with templates.